test/boost/gcp_object_storage_test.cc

```diff
@@ -326,131 +83,134 @@ static future<> create_object_of_size(storage::client& c, std::string_view bucke
+        auto infos = co_await c.list_objects(env.bucket, name);
+        BOOST_REQUIRE(std::find_if(infos.begin(), infos.end(), [&](auto& info) {
+            return info.name == name;
+        }) != infos.end());
+    }
 
-        co_await c.delete_object(env.bucket, name);
+    co_await c.delete_object(env.bucket, name);
 
-        auto infos = co_await c.list_objects(env.bucket, name);
-        BOOST_REQUIRE(infos.empty());
-    });
+    auto infos = co_await c.list_objects(env.bucket, name);
+    BOOST_REQUIRE(infos.empty());
 }
 
-SEASTAR_TEST_CASE(test_gcp_storage_skip_read, *check_gcp_storage_test_enabled()) {
-    return gcp_test_helper([](const tmpdir& tmp, storage::client& c, gcp_test_env& env) -> future<> {
-        auto name = fmt::format("{}", utils::UUID_gen::get_time_UUID());
-        std::vector<temporary_buffer<char>> bufs;
-        constexpr size_t file_size = 12*1024*1024 + 384*7 + 31;
-
-        co_await create_object_of_size(c, env.bucket, name, 12*1024*1024, &bufs);
-        for (size_t i = 0; i < 20; ++i) {
-            auto source = c.create_download_source(env.bucket, name);
-            auto copy = bufs | std::views::transform([](auto& buf) { return buf.share(); });
-            auto is1 = seastar::input_stream<char>(std::move(source));
-            auto is2 = seastar::input_stream<char>(create_memory_source(std::vector<temporary_buffer<char>>(copy.begin(), copy.end())));
-
-            std::exception_ptr p;
-            try {
-
-                size_t pos = 0; 
-                while (pos < file_size) {
-                    auto rem = file_size - pos;
-                    auto skip = tests::random::get_int(std::min(rem, size_t(100)), rem);
-                    auto read = std::min(rem - skip, size_t(tests::random::get_int(31, 2048)));
-                    co_await is1.skip(skip);
-                    co_await is2.skip(skip);
-
-                    auto b1 = co_await is1.read_exactly(read);
-                    auto b2 = co_await is2.read_exactly(read);
-
-                    BOOST_REQUIRE_EQUAL(b1.size(), b2.size());
-                    if (b1 != b2) {
-                        BOOST_TEST_MESSAGE(fmt::format("diff at {} ({} bytes)", pos + skip, read));
-                        auto i = std::mismatch(b1.begin(), b1.end(), b2.begin());
-                        BOOST_TEST_MESSAGE(fmt::format("offset {}", std::distance(b1.begin(), i.first)));
-                    }
-                    BOOST_REQUIRE_EQUAL(b1, b2);
-                    pos += (skip + read);
+SEASTAR_FIXTURE_TEST_CASE(test_gcp_storage_skip_read, local_gcs_wrapper, *check_gcp_storage_test_enabled()) {
+    auto& env = *this;
+    auto& c = env.client();
+    auto name = fmt::format("{}", utils::UUID_gen::get_time_UUID());
+    std::vector<temporary_buffer<char>> bufs;
+    constexpr size_t file_size = 12*1024*1024 + 384*7 + 31;
+
+    co_await create_object_of_size(c, env.bucket, name, 12*1024*1024, &bufs);
+    for (size_t i = 0; i < 20; ++i) {
```

Comment:
It would be nice if this patch didn't touch the indentation of not changed lines, but fixed them with a followup patch. Would be _much_ easier to review

Reply:
Perhaps. But it would also mean leaving a bunch of code with broken indentation, which makes the code being reviewed harder to read. Pros, cons...

Reply:
> But it would also mean leaving a bunch of code with broken indentation, which makes the code being reviewed harder to read 

With the commit fixing indentation afterwards, it's not a problem. While-reviewing it could be a problem, but my experience 

> Pros, cons...

This consensus is documented in coding style (of seastar)

> When making a change, if you need to insert an indentation level, you can temporarily break the rules by insering a half-indent, so that the patch is easily reviewable:
> ...
> A follow-up patch can restore the indents without any functional changes.